includes/global-settings-komoju.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+/*
+ * Settings for the whole Komoju plugin
+ */
+
+return [
+    [
+        'title'       => 'API Settings',
+        'type'        => 'title',
+        'id'          => 'api-settings-in-komoju',
+        'description' => sprintf(__('Default url for the webhook is %s. Use this if you\'re not sure what it should be.', 'komoju-woocommerce'), $this->url_for_webhooks()),
```

Comment:
Hmm, the description is not showing (screenshot from master branch):

![Screen Shot 2022-05-23 at 11 05 36](https://user-images.githubusercontent.com/16765206/169730024-067d7a4e-f80c-40b4-b774-299c6c2c0590.png)

Reply:
It looks like gateway settings is based on [`WC_Settings_API`](https://hooks.wbcomdesigns.com/reference/classes/wc_settings_api/), but regular settings pages don't actually use that interface...!

I'll have to write some custom code to show these.